tests/serialize/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I think that keeping the serialize scripts out of the top level `package.json` is orthogonal to whether there should be a `tests/serialize/package.json` file. It would be possible, for example, to use a tool like gulp or grunt, or a regular old JS file using node APIs to run scripts from the `tests/serialize/` subdirectory.

The reason I'm pointing this out is that,  far as I can tell, having a `package.json` file at the root of a directory is what defines that directory as an npm package. Since the `tests/serialize` subdirectory is not in fact an `npm` package, I'm hesitant to introduce another `package.json`.